target/linux/ramips/dts/mt7621_iptime_a3002me.dts

```diff
@@ -0,0 +1,151 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+	compatible = "iptime,a3002me", "mediatek,mt7621-soc";
```

Comment:
No, this should be `iptime,a3002mesh`.

Reply:
But stock uImage introduce self as "a3002me". I'm little bit confused.

![image](https://user-images.githubusercontent.com/1674267/153390259-eddd3b86-2112-4e11-988b-25f921660824.png)

Reply:
https://download.iptime.co.kr/online_upgrade/a3002me_ml_14_124.bin

Reply:
This string is for OpenWrt, not iptime code name. You should keep the `UIMAGE_NAME`, of course.